Lab1.ipynb

```diff
@@ -0,0 +1,482 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# A little statistis Q1B\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 143,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import matplotlib\n",
+    "import matplotlib.pyplot as plt\n",
+    "import scipy\n",
+    "from scipy import stats"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 144,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "0.8413447460685429\n",
+      "0.9986501019683699\n",
+      "0.0013498980316301035\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Integrate the standard normal distribution\n",
+    "prob1 = stats.norm.cdf(1)\n",
+    "prob2 = stats.norm.cdf(3)\n",
+    "prob3 = 1- stats.norm.cdf(3)\n",
+    "\n",
+    "print(prob1)\n",
+    "print(prob2)\n",
+    "print(prob3)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The probabilities match the cumulative z values.\n",
+    "Z = 1 => 0.84134\n",
+    "Z = 3 => 0.99865\n",
+    "Z = -3 => 0.00003\n"
```

Comment:
I believe the probability for z = -3 is 0.0013498980316301035 as shown by your third output above

Reply:
Thanks! It seems like I copy pasted the wrong value.